docs/topics/add-rule-to-mta.adoc

```diff
@@ -14,5 +14,5 @@ A {ProductName} rule is installed by copying the rule to the appropriate {Produc
 +
 [NOTE]
 ====
-In a Windows operating system, the rules are located in `\Documents and Settings&#x5c;<USER_NAME>\.mta\rules\` or `\Users&#x5c;<USER_NAME>\.mta\rules\`.
+In a Windows operating system, the rules are located in `\Documents and Settings&#x5c;<USER_NAME>\.{LC_PSN}\rules\` or `\Users&#x5c;<USER_NAME>\.{LC_PSN}\rules\`.
```

Comment:
What are these weird extended Ascii characters? Attributes normally do not render in backticks. YOu need "++"